
time="2025-07-30T23:27:34-03:00" level=warning msg="/home/rinha-user/rinha-de-backend-2025/participantes/Maxel-Uds-quarkus/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion"
 api01 Pulling 
 load_balancer Pulling 
 api02 Pulling 
 fcce199ebec0 Already exists 
 4f4fb700ef54 Already exists 
 42c077c10790 Already exists 
 992fa0855102 Already exists 
 ec661f466405 Already exists 
 94ad8e5864b2 Already exists 
 b76405cb56cd Already exists 
 9bfcb21bb22b Already exists 
 74da253cf0ae Pulling fs layer 
 c5d2055a8cf0 Pulling fs layer 
 a2b54f0669eb Pulling fs layer 
 59fcc6ae1197 Pulling fs layer 
 be9f76185eda Pulling fs layer 
 87c67c69533c Pulling fs layer 
 b3a875eb05ae Pulling fs layer 
 87c67c69533c Waiting 
 59fcc6ae1197 Waiting 
 be9f76185eda Waiting 
 b3a875eb05ae Waiting 
 fcce199ebec0 Already exists 
 4f4fb700ef54 Already exists 
 ec661f466405 Already exists 
 b76405cb56cd Already exists 
 74da253cf0ae Pulling fs layer 
 c5d2055a8cf0 Pulling fs layer 
 a2b54f0669eb Pulling fs layer 
 74da253cf0ae Verifying Checksum 
 74da253cf0ae Downloading [==================================================>]      94B/94B
 74da253cf0ae Verifying Checksum 
 74da253cf0ae Download complete 
 74da253cf0ae Download complete 
 74da253cf0ae Extracting [==================================================>]      94B/94B
 74da253cf0ae Extracting [==================================================>]      94B/94B
 74da253cf0ae Extracting [==================================================>]      94B/94B
 74da253cf0ae Extracting [==================================================>]      94B/94B
 a2b54f0669eb Downloading [>                                                  ]  303.1kB/30.01MB
 a2b54f0669eb Downloading [>                                                  ]  303.1kB/30.01MB
 74da253cf0ae Pull complete 
 74da253cf0ae Pull complete 
 c5d2055a8cf0 Downloading [==================================================>]     102B/102B
 c5d2055a8cf0 Verifying Checksum 
 c5d2055a8cf0 Download complete 
 c5d2055a8cf0 Downloading [==================================================>]     102B/102B
 c5d2055a8cf0 Verifying Checksum 
 c5d2055a8cf0 Download complete 
 c5d2055a8cf0 Extracting [==================================================>]     102B/102B
 c5d2055a8cf0 Extracting [==================================================>]     102B/102B
 c5d2055a8cf0 Extracting [==================================================>]     102B/102B
 c5d2055a8cf0 Extracting [==================================================>]     102B/102B
 a2b54f0669eb Downloading [============>                                      ]  7.696MB/30.01MB
 a2b54f0669eb Downloading [============>                                      ]  7.696MB/30.01MB
 c5d2055a8cf0 Pull complete 
 c5d2055a8cf0 Pull complete 
 a2b54f0669eb Downloading [==============================>                    ]  18.51MB/30.01MB
 a2b54f0669eb Downloading [==============================>                    ]  18.51MB/30.01MB
 a2b54f0669eb Download complete 
 a2b54f0669eb Download complete 
 a2b54f0669eb Extracting [>                                                  ]  327.7kB/30.01MB
 a2b54f0669eb Extracting [>                                                  ]  327.7kB/30.01MB
 a2b54f0669eb Extracting [==========>                                        ]  6.226MB/30.01MB
 a2b54f0669eb Extracting [==========>                                        ]  6.226MB/30.01MB
 a2b54f0669eb Extracting [====================>                              ]  12.45MB/30.01MB
 a2b54f0669eb Extracting [====================>                              ]  12.45MB/30.01MB
 a2b54f0669eb Extracting [===============================>                   ]  18.68MB/30.01MB
 a2b54f0669eb Extracting [===============================>                   ]  18.68MB/30.01MB
 a2b54f0669eb Extracting [========================================>          ]  24.25MB/30.01MB
 a2b54f0669eb Extracting [========================================>          ]  24.25MB/30.01MB
 a2b54f0669eb Extracting [==================================================>]  30.01MB/30.01MB
 a2b54f0669eb Extracting [==================================================>]  30.01MB/30.01MB
 a2b54f0669eb Pull complete 
 a2b54f0669eb Pull complete 
 api01 Pulled 
 api02 Pulled 
 59fcc6ae1197 Downloading [==================================================>]     828B/828B
 59fcc6ae1197 Verifying Checksum 
 59fcc6ae1197 Download complete 
 59fcc6ae1197 Extracting [==================================================>]     828B/828B
 59fcc6ae1197 Extracting [==================================================>]     828B/828B
 59fcc6ae1197 Pull complete 
 87c67c69533c Downloading [>                                                  ]  2.376kB/233.3kB
 87c67c69533c Download complete 
 be9f76185eda Downloading [>                                                  ]  527.6kB/119.9MB
 be9f76185eda Downloading [====>                                              ]  10.09MB/119.9MB
 be9f76185eda Downloading [========>                                          ]  19.64MB/119.9MB
 be9f76185eda Downloading [===========>                                       ]   27.6MB/119.9MB
 be9f76185eda Downloading [===============>                                   ]  37.67MB/119.9MB
 be9f76185eda Downloading [====================>                              ]  48.29MB/119.9MB
 be9f76185eda Downloading [=========================>                         ]  60.51MB/119.9MB
 b3a875eb05ae Downloading [==================================================>]     439B/439B
 b3a875eb05ae Download complete 
 be9f76185eda Downloading [============================>                      ]  68.98MB/119.9MB
 be9f76185eda Downloading [============================>                      ]  69.51MB/119.9MB
 be9f76185eda Downloading [=============================>                     ]  70.57MB/119.9MB
 be9f76185eda Downloading [==============================>                    ]   72.7MB/119.9MB
 be9f76185eda Downloading [================================>                  ]  76.94MB/119.9MB
 be9f76185eda Downloading [===================================>               ]  84.36MB/119.9MB
 be9f76185eda Downloading [========================================>          ]   96.1MB/119.9MB
 be9f76185eda Downloading [============================================>      ]  107.9MB/119.9MB
 be9f76185eda Downloading [=================================================> ]  118.6MB/119.9MB
 be9f76185eda Verifying Checksum 
 be9f76185eda Download complete 
 be9f76185eda Extracting [>                                                  ]  557.1kB/119.9MB
 be9f76185eda Extracting [==>                                                ]  6.128MB/119.9MB
 be9f76185eda Extracting [====>                                              ]  10.58MB/119.9MB
 be9f76185eda Extracting [=======>                                           ]  17.27MB/119.9MB
 be9f76185eda Extracting [========>                                          ]  20.61MB/119.9MB
 be9f76185eda Extracting [========>                                          ]  21.17MB/119.9MB
 be9f76185eda Extracting [=========>                                         ]  22.28MB/119.9MB
 be9f76185eda Extracting [============>                                      ]  29.52MB/119.9MB
 be9f76185eda Extracting [==============>                                    ]  33.98MB/119.9MB
 be9f76185eda Extracting [=================>                                 ]  41.22MB/119.9MB
 be9f76185eda Extracting [====================>                              ]  49.02MB/119.9MB
 be9f76185eda Extracting [=====================>                             ]  51.25MB/119.9MB
 be9f76185eda Extracting [======================>                            ]  55.15MB/119.9MB
 be9f76185eda Extracting [=========================>                         ]  60.72MB/119.9MB
 be9f76185eda Extracting [============================>                      ]  69.07MB/119.9MB
 be9f76185eda Extracting [===============================>                   ]  75.76MB/119.9MB
 be9f76185eda Extracting [==================================>                ]  81.89MB/119.9MB
 be9f76185eda Extracting [====================================>              ]  87.46MB/119.9MB
 be9f76185eda Extracting [=====================================>             ]   90.8MB/119.9MB
 be9f76185eda Extracting [======================================>            ]  91.91MB/119.9MB
 be9f76185eda Extracting [=======================================>           ]  95.26MB/119.9MB
 be9f76185eda Extracting [========================================>          ]  96.93MB/119.9MB
 be9f76185eda Extracting [=========================================>         ]  99.16MB/119.9MB
 be9f76185eda Extracting [=========================================>         ]  99.71MB/119.9MB
 be9f76185eda Extracting [=========================================>         ]  100.3MB/119.9MB
 be9f76185eda Extracting [==========================================>        ]  101.4MB/119.9MB
 be9f76185eda Extracting [==========================================>        ]  103.1MB/119.9MB
 be9f76185eda Extracting [===========================================>       ]  104.7MB/119.9MB
 be9f76185eda Extracting [================================================>  ]    117MB/119.9MB
 be9f76185eda Extracting [==================================================>]  119.9MB/119.9MB
 be9f76185eda Pull complete 
 87c67c69533c Extracting [=======>                                           ]  32.77kB/233.3kB
 87c67c69533c Extracting [==================================================>]  233.3kB/233.3kB
 87c67c69533c Extracting [==================================================>]  233.3kB/233.3kB
 87c67c69533c Pull complete 
 b3a875eb05ae Extracting [==================================================>]     439B/439B
 b3a875eb05ae Extracting [==================================================>]     439B/439B
 b3a875eb05ae Pull complete 
 load_balancer Pulled 
 Network maxel-uds-quarkus_payment-router  Creating
 Network maxel-uds-quarkus_payment-router  Created
 Container postgres  Creating
 Container postgres  Created
 Container api1  Creating
 Container api2  Creating
 Container api1  Created
 Container api2  Created
 Container load_balancer  Creating
 Container load_balancer  Created
Attaching to api1, api2, load_balancer, postgres
postgres       | The files belonging to this database system will be owned by user "postgres".
postgres       | This user must also own the server process.
postgres       | 
postgres       | The database cluster will be initialized with locale "en_US.utf8".
postgres       | The default database encoding has accordingly been set to "UTF8".
postgres       | The default text search configuration will be set to "english".
postgres       | 
postgres       | Data page checksums are disabled.
postgres       | 
postgres       | fixing permissions on existing directory /var/lib/postgresql/data ... ok
postgres       | creating subdirectories ... ok
postgres       | selecting dynamic shared memory implementation ... posix
postgres       | selecting default max_connections ... 100
postgres       | selecting default shared_buffers ... 128MB
postgres       | selecting default time zone ... Etc/UTC
postgres       | creating configuration files ... ok
postgres       | running bootstrap script ... ok
postgres       | performing post-bootstrap initialization ... ok
postgres       | syncing data to disk ... ok
postgres       | 
postgres       | 
postgres       | Success. You can now start the database server using:
postgres       | 
postgres       |     pg_ctl -D /var/lib/postgresql/data -l logfile start
postgres       | 
postgres       | initdb: warning: enabling "trust" authentication for local connections
postgres       | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
postgres       | waiting for server to start....2025-07-31 02:27:53.102 UTC [48] LOG:  starting PostgreSQL 16.9 (Debian 16.9-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres       | 2025-07-31 02:27:53.107 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres       | 2025-07-31 02:27:53.126 UTC [51] LOG:  database system was shut down at 2025-07-31 02:27:52 UTC
postgres       | 2025-07-31 02:27:53.139 UTC [48] LOG:  database system is ready to accept connections
postgres       |  done
postgres       | server started
postgres       | CREATE DATABASE
postgres       | 
postgres       | 
postgres       | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
postgres       | psql:/docker-entrypoint-initdb.d/init.sql:1: NOTICE:  table "payments" does not exist, skipping
postgres       | DROP TABLE
postgres       | CREATE TABLE
postgres       | 
postgres       | 
postgres       | waiting for server to shut down....2025-07-31 02:27:53.924 UTC [48] LOG:  received fast shutdown request
postgres       | 2025-07-31 02:27:53.932 UTC [48] LOG:  aborting any active transactions
postgres       | 2025-07-31 02:27:54.002 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
postgres       | 2025-07-31 02:27:54.003 UTC [49] LOG:  shutting down
postgres       | 2025-07-31 02:27:54.008 UTC [49] LOG:  checkpoint starting: shutdown immediate
postgres       | 2025-07-31 02:27:54.133 UTC [49] LOG:  checkpoint complete: wrote 922 buffers (5.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.050 s, sync=0.052 s, total=0.131 s; sync files=303, longest=0.031 s, average=0.001 s; distance=4263 kB, estimate=4263 kB; lsn=0/19140B0, redo lsn=0/19140B0
postgres       | 2025-07-31 02:27:54.141 UTC [48] LOG:  database system is shut down
postgres       |  done
postgres       | server stopped
postgres       | 
postgres       | PostgreSQL init process complete; ready for start up.
postgres       | 
postgres       | 2025-07-31 02:27:54.261 UTC [1] LOG:  starting PostgreSQL 16.9 (Debian 16.9-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres       | 2025-07-31 02:27:54.261 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres       | 2025-07-31 02:27:54.261 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres       | 2025-07-31 02:27:54.307 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres       | 2025-07-31 02:27:54.326 UTC [66] LOG:  database system was shut down at 2025-07-31 02:27:54 UTC
postgres       | 2025-07-31 02:27:54.337 UTC [1] LOG:  database system is ready to accept connections
load_balancer  | 2025/07/31 02:28:05 [error] 7#7: *3 open() "/usr/local/openresty/nginx/html/purge-payments" failed (2: No such file or directory), client: 172.20.0.1, server: , request: "POST /purge-payments HTTP/1.1", host: "localhost:9999"
[Kload_balancer exited with code 0
api2           | __  ____  __  _____   ___  __ ____  ______ 
api2           |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api2           |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api2           | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api2           | 2025-07-31 02:29:17,257 ERROR [org.jbo.thr.errors] (executor-thread-3) Thread Thread[#38,executor-thread-3,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,257 ERROR [org.jbo.thr.errors] (executor-thread-6) Thread Thread[#41,executor-thread-6,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,257 ERROR [org.jbo.thr.errors] (executor-thread-10) Thread Thread[#45,executor-thread-10,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,257 ERROR [org.jbo.thr.errors] (executor-thread-13) Thread Thread[#48,executor-thread-13,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,258 ERROR [org.jbo.thr.errors] (executor-thread-4) Thread Thread[#39,executor-thread-4,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,258 ERROR [org.jbo.thr.errors] (executor-thread-1) Thread Thread[#36,executor-thread-1,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,258 ERROR [org.jbo.thr.errors] (executor-thread-11) Thread Thread[#46,executor-thread-11,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,258 ERROR [org.jbo.thr.errors] (executor-thread-15) Thread Thread[#50,executor-thread-15,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,258 ERROR [org.jbo.thr.errors] (executor-thread-12) Thread Thread[#47,executor-thread-12,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,258 ERROR [org.jbo.thr.errors] (executor-thread-14) Thread Thread[#49,executor-thread-14,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,259 ERROR [org.jbo.thr.errors] (executor-thread-8) Thread Thread[#43,executor-thread-8,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,259 ERROR [org.jbo.thr.errors] (executor-thread-7) Thread Thread[#42,executor-thread-7,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,259 ERROR [org.jbo.thr.errors] (executor-thread-2) Thread Thread[#37,executor-thread-2,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,259 ERROR [org.jbo.thr.errors] (executor-thread-5) Thread Thread[#40,executor-thread-5,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-31 02:29:17,259 ERROR [org.jbo.thr.errors] (executor-thread-9) Thread Thread[#44,executor-thread-9,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api1           | __  ____  __  _____   ___  __ ____  ______ 
api1           |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api1           |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api1           | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api1           | 2025-07-31 02:29:17,270 ERROR [org.jbo.thr.errors] (executor-thread-10) Thread Thread[#45,executor-thread-10,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,270 ERROR [org.jbo.thr.errors] (executor-thread-13) Thread Thread[#48,executor-thread-13,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,270 ERROR [org.jbo.thr.errors] (executor-thread-3) Thread Thread[#38,executor-thread-3,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,270 ERROR [org.jbo.thr.errors] (executor-thread-2) Thread Thread[#37,executor-thread-2,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,270 ERROR [org.jbo.thr.errors] (executor-thread-5) Thread Thread[#40,executor-thread-5,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,270 ERROR [org.jbo.thr.errors] (executor-thread-8) Thread Thread[#43,executor-thread-8,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,270 ERROR [org.jbo.thr.errors] (executor-thread-9) Thread Thread[#44,executor-thread-9,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,270 ERROR [org.jbo.thr.errors] (executor-thread-7) Thread Thread[#42,executor-thread-7,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,271 ERROR [org.jbo.thr.errors] (executor-thread-15) Thread Thread[#50,executor-thread-15,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,274 ERROR [org.jbo.thr.errors] (executor-thread-11) Thread Thread[#46,executor-thread-11,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,274 ERROR [org.jbo.thr.errors] (executor-thread-12) Thread Thread[#47,executor-thread-12,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,274 ERROR [org.jbo.thr.errors] (executor-thread-14) Thread Thread[#49,executor-thread-14,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,274 ERROR [org.jbo.thr.errors] (executor-thread-6) Thread Thread[#41,executor-thread-6,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,270 ERROR [org.jbo.thr.errors] (executor-thread-4) Thread Thread[#39,executor-thread-4,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-31 02:29:17,274 ERROR [org.jbo.thr.errors] (executor-thread-1) Thread Thread[#36,executor-thread-1,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
[Kapi2 exited with code 143
[Kapi1 exited with code 143
postgres       | 2025-07-31 02:29:17.999 UTC [1] LOG:  received fast shutdown request
postgres       | 2025-07-31 02:29:18.004 UTC [1] LOG:  aborting any active transactions
postgres       | 2025-07-31 02:29:18.006 UTC [1] LOG:  background worker "logical replication launcher" (PID 69) exited with exit code 1
postgres       | 2025-07-31 02:29:18.008 UTC [64] LOG:  shutting down
postgres       | 2025-07-31 02:29:18.013 UTC [64] LOG:  checkpoint starting: shutdown immediate
postgres       | 2025-07-31 02:29:18.108 UTC [64] LOG:  checkpoint complete: wrote 406 buffers (2.5%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.024 s, sync=0.045 s, total=0.100 s; sync files=24, longest=0.037 s, average=0.002 s; distance=4926 kB, estimate=4926 kB; lsn=0/1DE3A68, redo lsn=0/1DE3A68
postgres       | 2025-07-31 02:29:18.112 UTC [1] LOG:  database system is shut down
[Kpostgres exited with code 0
log truncated at line 1000
